### ISSUE TITLE
Unable to run RADI with GPU acceleration

### ISSUE THREAD

**Issue by pariaspe** [JdeRobot/RoboticsAcademy#1520]
I'm getting this error with VGL when I'm trying to run the latest RADI with GPU acceleration:
```
[VGL] ERROR: Could not open display /dev/dri/card0.
[WARN] [1642851485.742479]: [GAZEBO] Check timeout exceeded
```

Since VGL fails, Gazebo launching also fails and I can not play with the exercise.
I was able to use GPU acceleration with RADI 3.1 versions.

**Comment by soumilbaldota** [comment]
You could try to do this and see if it works.
```sudo nvidia-xconfig -a --allow-empty-initial-configuration --use-display-device=None \ --virtual=1920x1200 --busid {busid}```

**Comment by soumilbaldota** [comment]
Similar to 
[link](https://github.com/VirtualGL/virtualgl/issues/108#issuecomment-531851472)

**Comment by soumilbaldota** [comment]
Similar to (https://github.com/VirtualGL/virtualgl/issues/108#issuecomment-531851472)

**Comment by pariaspe** [comment]
Hi @soumilbaldota, thanks for the suggestion but it didn't work in my system. Have you tried it?

**Comment by pariaspe** [comment]
Solved in #1902. Thanks @iamarkaj !
